vmdb/lib/tasks/test_metrics.rake

```diff
@@ -4,7 +4,7 @@
 #   original here: https://github.com/ManageIQ/metric_fu/blob/master/tasks/metric_fu.rake
 require_relative "./evm_test_helper"
 
-if defined?(RSpec)
+if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
```

Comment:
I was looking for RSpec and it wasn't used because the child tasks it calls need it.  It wasn't immediately obvious why it was like that.  I'm not saying what I suggested was better, I was hoping to make it more obvious what's going on by not loading the test related tasks by default if rspec isn't loaded.